__tests__/test-utils/index.js

```diff
@@ -1,8 +1,8 @@
 // Sun Aug 04 2019 19:34:56 GMT-0700
 const DEFAULT_TEST_TIME = Date.UTC(2019, 7, 5, 2, 34, 56, 78)
 
-export function timeoutPromise (ms) {
-  return new Promise((resolve, reject) => {
+export function timeoutPromise(ms: number): Promise<void> {
```

Comment:
Now that you have added Typescript types, rename this file to `index.ts`. With that I think the failing `yarn unit` step in the checks should pass.

Reply:
Ok, thank you! Was wondering what was causing those